graph/common/Storage.java

```diff
@@ -109,6 +113,15 @@ default Schema asSchema() {
 
             private final Integer fixedStartBytes;
 
+            public static Partition fromName(String name) {
+                if (name.equals(DEFAULT.name())) return DEFAULT;
+                else if (name.equals(VARIABLE_START_EDGE.name())) return VARIABLE_START_EDGE;
+                else if (name.equals(FIXED_START_EDGE.name())) return FIXED_START_EDGE;
+                else if (name.equals(OPTIMISATION_EDGE.name())) return OPTIMISATION_EDGE;
+                else if (name.equals(STATISTICS.name())) return STATISTICS;
+                else throw TypeDBException.of(UNRECOGNISED_VALUE);
+            }
```

Comment:
Use a int or even better a short to save space in the raft log and network

Reply:
This has been solved as per the last commit.